packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -489,14 +489,60 @@ export class SCWSigner implements Signer {
       standardErrors.rpc.internal(`client not found for chainId ${this.chain.id}`)
     );
 
+    const globalAccountAddress = this.accounts.find(
+      (account) => account.toLowerCase() !== subAccount.address.toLowerCase()
+    );
+
+    assertPresence(
+      globalAccountAddress,
+      standardErrors.provider.unauthorized('no global account found')
+    );
+
     const { request: subAccountRequest } = await createSubAccountSigner({
       address: subAccount.address,
       owner: ownerAccount.account,
       client: client,
       factory: subAccount.factory,
       factoryData: subAccount.factoryData,
+      parentAddress: globalAccountAddress,
```

Comment:
nit: we should standardize naming for the global account. No strong preference global, parent, universal are all good with me